contact-page.js

```diff
@@ -0,0 +1,116 @@
+var firstName = document.getElementById("first-name").value;
+var lastName = document.getElementById("last-name").value;
+var email = document.getElementById("email").value;
+var compName = document.getElementById("company-name").value;
+var compUrl = document.getElementById("url").value;
+var phone = document.getElementById("phone").value;
+var message = document.getElementById("comment").value;
+var errorText = document.getElementById("error-msg");
+var text;
+
+errorText.style.padding = "10px";
+
+firstName.addEventListener("blur", fstNameVerify, true);
+lastName.addEventListener("blur", lstNameVerify, true);
+email.addEventListener("blur", emailVerify, true);
+compName.addEventListener("blur", compNameVerify, true);
+compUrl.addEventListener("blur", compUrlVerify, true);
+phone.addEventListener("blur", phoneVerify, true);
+message.addEventListener("blur", messageVerify, true);
+
+function validateForm() {
+    if (firstName.value == "" || firstName.length < 3) {
+        firstName.style.border = "1px solid red";
+        text = "Please Enter A Valid First Name";
+        errorText.innerHTML = text;
+        firstName.focus();
+        return false;
+    } else {
+        return true;

```

Comment:
If the firstname field is valid, the function will stop here with this `return` and nothing else will be executed.